### PR TITLE
Prepare for registering in General

### PR DESCRIPTION
**Pull request by amellnik**
* Enable codecov

### CODE REVIEW

**Comment by amellnik** [comment]
I don't think there's actually much to do here unless we want to expand the docs -- this should be good to go.

**Comment by ajkeller34** [comment]
Closing and reopening to retrigger CI—one of the workers timed out before even starting the tests.

File: src/ansys.jl [hunk 1]
```diff
@@ -41,6 +41,7 @@ matrix elements. We expect these to be strings `parse(Float64, _)` can handle.
 =#
 
 using LinearAlgebra: diagind
+using Compat
```
Comment (dangirsh) [inline src/ansys.jl]: Curious why this was needed

Comment (ajkeller34) [reply src/ansys.jl]: Turns out `isnothing` wasn’t implemented yet in julia 1.0

**Pull request merged**